grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -665,22 +666,22 @@ class ProductAvailabilityService {
 						left outer join pa.inventoryItem ii
 						left outer join pa.binLocation bl
 						where pa.location = :location
-						and pa.product in (:products)
-						""", [location: location, products: products])
-            def status = { quantity -> quantity > 0 ? "inStock" : "outOfStock" }
```

Comment:
I removed this at all, because it has not been used anywhere: we were doing this:
```groovy
 [
    status                      : status(quantity),
    product                     : it[0],
    inventoryItem               : inventoryItem,
    binLocation                 : binLocation,
    quantityOnHand              : quantity,
    quantityAvailableToPromise  : it[4]
]
and then immedietaly mapping this to `new AvailableItem(...)`
So I also removed the `product` property, because of the same reason